instrumentation/gwt-2.0/javaagent/gwt-2.0-javaagent.gradle

```diff
@@ -45,11 +45,6 @@ dependencies {
   testImplementation group: 'org.eclipse.jetty', name: 'jetty-webapp', version: '9.4.35.v20201120'
 }
 
-task copyTestWebapp(type: Copy) {
```

Comment:
@laurit This wasn't needed right?

Reply:
It should be needed and test fails without this for me.

Reply:
Hmm - myself nor CI could reproduce any failure. Since compileGwt builds the war, what does this task do?

Reply:
This task copies greeting.html and WEB-INF/web.xml to war that is deployed by jetty. Without these files the application shouldn't really work because greeting.html is the page that we use for testing and if it is missing there is nothing to test. It is strange that it passed in ci, maybe it cached the test result and didn't really run the test? Anyway seems to have started to fail now https://github.com/open-telemetry/opentelemetry-java-instrumentation/runs/2225667052